views/default/css/elements/misc.php

```diff
@@ -59,3 +59,5 @@
 	display: none;
 	padding: 0;
 }
+
+<?php echo elgg_view('css/elements/spinner.css') ?>
```

Comment:
@ewinslow had brought up the idea of separating CSS into more files. For BC those would be pulled in from the original css/elgg/elements/*.php files, so this was for consistency with that plan I guess